content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2858,12 +2858,11 @@ model Post {
 
 <AlgoliaTerm name="apiReference" value="@ignore" />
 
-In [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and later, Prisma adds `@ignore` to fields that _refer to_ invalid models when you introspect.
+Add `@ignore` to a field that you want to exclude from the Prisma Client (for example, a field that you do not want Prisma users to update). Ignored fields are excluded from the generated Prisma Client. The model's `create` method is disabled when doing this for _required_ fields with no `@default` (as the database will not be able to create an entry without that data).
```

Comment:
```suggestion
Add `@ignore` to a field that you want to exclude from the Prisma Client (for example, a field that you do not want Prisma users to update). Ignored fields are excluded from the generated Prisma Client. The model's `create` method is disabled when doing this for _required_ fields with no `@default` (because the database cannot create an entry without that data).
```